### ISSUE TITLE
Telescope classic theme is not activated

### ISSUE THREAD

**Issue by basava70** [AlexvZyl/nordic.nvim#54]
Hey man, loved your theme. My current favorite theme of nord palette.
I tried to get the classic theme for telescope popup but the theme seems to be in flat.
Do you have any suggestion regarding this ?

**Comment by AlexvZyl** [comment]
Glad you like it!  Can you please show me your config?

**Comment by basava70** [comment]
> Glad you like it! Can you please show me your config?

require 'nordic' .setup {
    -- Available themes: 'nordic', 'onedark'.
    -- Onedark is WIP.
    theme = 'nordic',
    -- Enable bold keywords.
    bold_keywords = false,
    -- Enable italic comments.
    italic_comments = true,
    -- Enable general editor background transparency.
    transparent_bg = false,
    -- Nordic specific options.
    -- Set all to false to use original Nord colors.
    -- Adjusts some colors to make the theme a bit nicer (imo).
    nordic = {
        -- Reduce the overall amount of blue in the theme (diverges from base Nord).
        reduced_blue = true,
    },
    -- Onedark specific options.
    -- Set all to false to keep original onedark colors.
    -- Adjusts some colors to make the theme a bit nicer (imo).
    -- WIP.
    onedark = {
        -- Brighten the whites to fit the theme better.
        brighter_whites = true,
    },
    -- Override the styling of any highlight group.
    override = {},
    cursorline = {
        -- Enable bold font in cursorline.
        bold = false,
        -- Avialable styles: 'dark', 'light'.
        theme = 'light',
        -- Hide the cursorline when the window is not focused.
        hide_unfocused = true,
    },
    noice = {
        -- Available styles: `classic`, `flat`.
        style = 'classic',
    },
    telescope = {
        -- Available styles: `classic`, `flat`.
        style = 'classic',
    },
    leap = {
        -- Dims the backdrop when using leap.
        dim_backdrop = false,
    },
}
Is this what you asked for ?

**Comment by basava70** [comment]
> Yes, that is what I asked for. I am not sure what is wrong. Could you share a screenshot of what you telescope looks like?

![Screenshot from 2023-04-04 11-20-42](https://user-images.githubusercontent.com/124213816/229747382-3789231d-b2a6-424f-8e5c-e3ac7cbc5e34.png)

**Comment by basava70** [comment]
I have the config file nordic.lua in after/plugin/nordic.lua and I have 
 use({ 'AlexvZyl/nordic.nvim',
  as = 'nordic',
  config = function()
    vim.cmd('colorscheme nordic')
  end
})

in lua/seshu_lua/packer.lua
I have used the same method as primeagen used. 
I did the same for other plugins as well. Add use ' repo/foo.nvim' in packer.lua
and put the config in after/plugin/foo.lua.

**Comment by AlexvZyl** [comment]
I am not sure what method ThePrimeagen used, but should it not look like this?

```lua
config = function()
    require 'nordic' .setup {}
    vim.cmd('colorscheme nordic')
end
```

**Comment by AlexvZyl** [comment]
What I suspect, is that `vim.cmd('colorscheme nordic')` is being called before `after/plugin/nordic.lua` is being sourced.  

Remove these lines:
```lua
config = function()
    vim.cmd('colorscheme nordic')
end
```

And add `vim.cmd('colorscheme nordic')` to the end of 'after/plugin/nordic.lua'.

**Comment by basava70** [comment]
> 



> ```lua
> vim.cmd('colorscheme nordic')
> ```


Nope, didnt work. I closed the entire nvim and reopened it multiple times. did PackerSync and :so too in both nordic.lua and packer.lua.

**Comment by AlexvZyl** [comment]
It has to be a sourcing issue, since I am sure that the config option works.  Please try this, just to check if I am right:

```lua
config = function()
    require 'nordic' .setup {
        telescope = {
            style = 'classic'
        }
    }
    vim.cmd 'colorscheme nordic'
end
```

**Comment by AlexvZyl** [comment]
I am officially confused...  I can't look more into it right now, but I would like to solve this problem.  At least flat doesn't look that bad hey ;)

**Comment by basava70** [comment]
> I am officially confused... I can't look more into it right now, but I would like to solve this problem. At least flat doesn't look that bad ;)

No, it doesnt. It looks good. But just like you, I have that itch :p why is it not working lol.

**Comment by mactep** [comment]
@basava70 can you "trim" your config to provide us a minimal config to reproduce this? Disabling every other plugin and removing every unnecessary option would be nice. You could also publish your dotfiles so we can clone and test locally.

**Comment by basava70** [comment]
> @basava70 can you "trim" your config to provide us a minimal config to reproduce this? Disabling every other plugin and removing every unnecessary option would be nice. You could also publish your dotfiles so we can clone and test locally.

Ok, I think you (@AlexvZyl ) can close this.  It is a sourcing issue. I just deleted everything and wrote the files again and it got fixed. I had to produce colorscheme nordic after the require 'nordic'.setup().
And @mactep  thanks for the help. for some reason the idea that  " I can trim everything down and construct from scratch" didnt come to me lol. 

Thanks for all the help guys really appreciate the nvim community. Cheers!!
